server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestDownloadByCounty.java

```diff
@@ -88,16 +83,13 @@ public String endpoint(final Request the_request, final Response the_response) {
           Main.GSON.toJson(matches, bw);
           bw.flush();
         } catch (final IOException e) {
-          status = HttpStatus.INTERNAL_SERVER_ERROR_500;
-          result = "Unable to stream response";
+          serverError(the_response, "Unable to stream response");
         }
       }
     } else {
-      status = HttpStatus.NOT_FOUND_404;
-      result = "Invalid county ID specified";
+      dataNotFound(the_response, "Invalid county ID specified");
```

Comment:
I have not updated or addressed any of the terminology used in log messages.  We should (eventually) rewrite them to strictly use the terminology used in the documentation of the various exceptional behavior functions in `AbstractEndpoint`, which matches that which is documented in the specification.